<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>175</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>175</width>
    <height>90</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>.</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>175</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>175</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Masaüstü/C++/eraser.png</normaloff>../../Masaüstü/C++/eraser.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>65</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Masaüstü/C++/check.png</normaloff>../../Masaüstü/C++/check.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>67</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Masaüstü/C++/settings.png</normaloff>../../Masaüstü/C++/settings.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Masaüstü/C++/refresh.png</normaloff>../../Masaüstü/C++/refresh.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>67</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Masaüstü/C++/information.png</normaloff>../../Masaüstü/C++/information.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>xzzzz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>67</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Masaüstü/C++/history.png</normaloff>../../Masaüstü/C++/history.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Masaüstü/C++/office-push-pin.png</normaloff>../../Masaüstü/C++/office-push-pin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>7</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>7</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_9</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
